integration/vite-css-dev-test.ts

```diff
@@ -60,28 +63,28 @@ test.describe("Vite CSS dev", () => {
             );
           }
         `,
-        "app/routes/_index/styles-bundled.css": css`
+        "app/styles-bundled.css": css`
           .index_bundled {
             background: papayawhip;
             padding: ${TEST_PADDING_VALUE};
           }
         `,
-        "app/routes/_index/styles-linked.css": css`
+        "app/styles-linked.css": css`
           .index_linked {
             background: salmon;
             padding: ${TEST_PADDING_VALUE};
           }
         `,
-        "app/routes/_index/styles.module.css": css`
+        "app/styles.module.css": css`
           .index {
             background: peachpuff;
             padding: ${TEST_PADDING_VALUE};
           }
         `,
-        "app/routes/_index/route.tsx": js`
-          import "./styles-bundled.css";
-          import linkedStyles from "./styles-linked.css?url";
-          import cssModulesStyles from "./styles.module.css";
+        "app/routes/_index.tsx": js`
```

Comment:
This is fixing an unrelated issue with the test while I'm here. The template already contains `app/routes/_index.tsx`, so we were adding a conflicting route with `app/routes/_index/route.tsx`.